pause/pause.c

```diff
@@ -0,0 +1,28 @@
+//
+// Copyright (c) 2017 Intel Corporation
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <signal.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <unistd.h>
+
+static void sigdown(int signo) {
+	psignal(signo, "shutting down, got signal");
+	exit(0);
+}
+
+int main() {
```

Comment:
A C pause binary is going to be smaller than a golang one, but this might (again) lead to problems supporting CentOS and RHEL...?:

- https://github.com/clearcontainers/runtime/pull/434

Reply:
Will `cgo` pause binary have same problem?

Reply:
@jodh-intel That is because we want to support multiple distro rootfs. IMO the guest rootfs should be minimal and contains as less packages as possible. There is little value in support different distros. In hyperstart, we simple throw in a few binaries (busybox, iptables, ipvsadm, mount.nfs etc.) for different architectures. And there is no distro support at all.

For kata, I understand the difficulty to add systemd support on our own and we can rely on a certain LTS distro. But I still don't see the value of support different distros in guest rootfs.

Reply:
@bergwolf The value is for users/customers that want to control what's in the guest and basically let their OSV support the guest image.

Reply:
I vote to supporting different guest images, this could be useful feature for us.